digdag-docs/src/command_reference.rst

```diff
@@ -299,6 +299,16 @@ Runs a digdag server. --memory or --database option is required. Examples:
 
   Example: ``--disable-executor-loop``
 
+:command:`-p, --param KEY=VALUE`
+  Add a session parameter (use multiple times to set many parameters) in KEY=VALUE syntax. This parameter is availabe using ``${...}`` syntax in the YAML file, or using language API.
+
+  Example: ``-p environment=staging``
+
+:command:`-P, --params-file PATH`
+  Read parameters from a YAML file. Nested parameter (like {mysql: {user: me}}) are accessible using "." syntax (like \${mysql.user}).
+
+  Example: ``-P params.dig``
```

Comment:
This file isn't a workflow definition file, right? So `params.yml` (or `.yaml`) in the example seems better to me.

Reply:
Thank you for your review! You are right, so I'll correct it. 

But, other parts of this documents also using `params.dig`, right? 
In order to fix those, I'll create another PR. Is this all right?

Reply:
Good catch. I think `params.dig` in docs should've been fixed in https://github.com/treasure-data/digdag/commit/b8dc67a9d5525125484cdc406e0a7efec47f1ff6.

Let's fix other `params.dig` that aren't related to this change in another PR :)

Reply:
Oh, I agree with you.  

Noted with thanks. After correcting this I'll also fix other `params.dig` in another PR ☺️